etc/api/shared.yaml

```diff
@@ -105,7 +105,7 @@ cargo:
 urls:
   authenticator_delegate: https://docs.rs/yup-oauth2/*/yup_oauth2/trait.AuthenticatorDelegate.html
 copyright:
-  license_abbrev: "MIT"
-  years: "2015-2020"
+  license_abbrev: MIT
+  years: 2015–2024
   authors:
-    - "Sebastian Thiel"
+    - Sebastian Thiel
```

Comment:
Maybe here there should be a line with blanked "Contributors"? What do you think?

Reply:
Sorry. Can you please say more? Where would you like it to appear?